Testsuite fails to build: error: reference to 'filesystem' is ambiguous

```
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/src/tests/test_filesystem.cc:37:21: error: reference to 'filesystem' is ambiguous
  BOOST_CHECK_EQUAL(filesystem::FileExists(filename), false);
                    ^
/usr/include/c++/v1/filesystem:267:1: note: candidate found by name lookup is 'std::filesystem'
_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM
^
/usr/include/c++/v1/__config:824:65: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM'
  _LIBCPP_BEGIN_NAMESPACE_STD inline namespace __fs { namespace filesystem {
                                                                ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/include/votca/tools/filesystem.h:27:11: note: candidate found by name lookup is 'votca::tools::filesystem'
namespace filesystem {
          ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/src/tests/test_filesystem.cc:44:21: error: reference to 'filesystem' is ambiguous
  BOOST_CHECK_EQUAL(filesystem::FileExists(filename2), true);
                    ^
/usr/include/c++/v1/filesystem:267:1: note: candidate found by name lookup is 'std::filesystem'
_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM
^
/usr/include/c++/v1/__config:824:65: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM'
  _LIBCPP_BEGIN_NAMESPACE_STD inline namespace __fs { namespace filesystem {
                                                                ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/include/votca/tools/filesystem.h:27:11: note: candidate found by name lookup is 'votca::tools::filesystem'
namespace filesystem {
          ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/src/tests/test_filesystem.cc:50:21: error: reference to 'filesystem' is ambiguous
  BOOST_CHECK_EQUAL(filesystem::GetFileExtension(filename), "txt");
                    ^
/usr/include/c++/v1/filesystem:267:1: note: candidate found by name lookup is 'std::filesystem'
_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM
^
/usr/include/c++/v1/__config:824:65: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM'
  _LIBCPP_BEGIN_NAMESPACE_STD inline namespace __fs { namespace filesystem {
                                                                ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/include/votca/tools/filesystem.h:27:11: note: candidate found by name lookup is 'votca::tools::filesystem'
namespace filesystem {
          ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/src/tests/test_filesystem.cc:53:21: error: reference to 'filesystem' is ambiguous
  BOOST_CHECK_EQUAL(filesystem::GetFileExtension(filename2), "");
                    ^
/usr/include/c++/v1/filesystem:267:1: note: candidate found by name lookup is 'std::filesystem'
_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM
^
/usr/include/c++/v1/__config:824:65: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM'
  _LIBCPP_BEGIN_NAMESPACE_STD inline namespace __fs { namespace filesystem {
                                                                ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/include/votca/tools/filesystem.h:27:11: note: candidate found by name lookup is 'votca::tools::filesystem'
namespace filesystem {
          ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/src/tests/test_filesystem.cc:56:21: error: reference to 'filesystem' is ambiguous
  BOOST_CHECK_EQUAL(filesystem::GetFileExtension(filename3), "gro");
                    ^
/usr/include/c++/v1/filesystem:267:1: note: candidate found by name lookup is 'std::filesystem'
_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM
^
/usr/include/c++/v1/__config:824:65: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM'
  _LIBCPP_BEGIN_NAMESPACE_STD inline namespace __fs { namespace filesystem {
                                                                ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/include/votca/tools/filesystem.h:27:11: note: candidate found by name lookup is 'votca::tools::filesystem'
namespace filesystem {
          ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/src/tests/test_filesystem.cc:62:21: error: reference to 'filesystem' is ambiguous
  BOOST_CHECK_EQUAL(filesystem::GetFileBase(filename), "bla");
                    ^
/usr/include/c++/v1/filesystem:267:1: note: candidate found by name lookup is 'std::filesystem'
_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM
^
/usr/include/c++/v1/__config:824:65: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM'
  _LIBCPP_BEGIN_NAMESPACE_STD inline namespace __fs { namespace filesystem {
                                                                ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/include/votca/tools/filesystem.h:27:11: note: candidate found by name lookup is 'votca::tools::filesystem'
namespace filesystem {
          ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/src/tests/test_filesystem.cc:65:21: error: reference to 'filesystem' is ambiguous
  BOOST_CHECK_EQUAL(filesystem::GetFileBase(filename2), "blubb");
                    ^
/usr/include/c++/v1/filesystem:267:1: note: candidate found by name lookup is 'std::filesystem'
_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM
^
/usr/include/c++/v1/__config:824:65: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM'
  _LIBCPP_BEGIN_NAMESPACE_STD inline namespace __fs { namespace filesystem {
                                                                ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/include/votca/tools/filesystem.h:27:11: note: candidate found by name lookup is 'votca::tools::filesystem'
namespace filesystem {
          ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/src/tests/test_filesystem.cc:68:21: error: reference to 'filesystem' is ambiguous
  BOOST_CHECK_EQUAL(filesystem::GetFileBase(filename3), "a.b.c.d");
                    ^
/usr/include/c++/v1/filesystem:267:1: note: candidate found by name lookup is 'std::filesystem'
_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM
^
/usr/include/c++/v1/__config:824:65: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_FILESYSTEM'
  _LIBCPP_BEGIN_NAMESPACE_STD inline namespace __fs { namespace filesystem {
                                                                ^
/disk-samsung/freebsd-ports/science/votca/work/votca-2022/tools/include/votca/tools/filesystem.h:27:11: note: candidate found by name lookup is 'votca::tools::filesystem'
namespace filesystem {
          ^
8 errors generated.
```

clang-12
FreeBSD 13

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -6,6 +6,7 @@ Version 2022.1 (released XX.01.22)
 
 -  fix PROJECT_VERSION in subdirs (#951) 
 -  make csg_resample and csg_stat spacing consistent (#956)
+-  fix build on FreeBSD (#959)
 
 Version 2022 (released 15.01.22)
 ================================

--- a/tools/src/tests/test_filesystem.cc
+++ b/tools/src/tests/test_filesystem.cc
@@ -34,38 +34,41 @@ BOOST_AUTO_TEST_SUITE(filesystem_test)
 BOOST_AUTO_TEST_CASE(FileExists) {
 
   std::string filename = "blablablablablabla.txt";
-  BOOST_CHECK_EQUAL(filesystem::FileExists(filename), false);
+  BOOST_CHECK_EQUAL(votca::tools::filesystem::FileExists(filename), false);
   std::string filename2 = "test_exist.txt";
   std::ofstream outfile(filename2);
 
   outfile << "my text here!" << std::endl;
 
   outfile.close();
-  BOOST_CHECK_EQUAL(filesystem::FileExists(filename2), true);
+  BOOST_CHECK_EQUAL(votca::tools::filesystem::FileExists(filename2), true);
 }
 
 BOOST_AUTO_TEST_CASE(GetFileExtension) {
 
   std::string filename = "bla.txt";
-  BOOST_CHECK_EQUAL(filesystem::GetFileExtension(filename), "txt");
+  BOOST_CHECK_EQUAL(votca::tools::filesystem::GetFileExtension(filename),
+                    "txt");
 
   std::string filename2 = "blubb";
-  BOOST_CHECK_EQUAL(filesystem::GetFileExtension(filename2), "");
+  BOOST_CHECK_EQUAL(votca::tools::filesystem::GetFileExtension(filename2), "");
 
   std::string filename3 = "a.b.c.d.gro";
-  BOOST_CHECK_EQUAL(filesystem::GetFileExtension(filename3), "gro");
+  BOOST_CHECK_EQUAL(votca::tools::filesystem::GetFileExtension(filename3),
+                    "gro");
 }
 
 BOOST_AUTO_TEST_CASE(GetFileBase) {
 
   std::string filename = "bla.txt";
-  BOOST_CHECK_EQUAL(filesystem::GetFileBase(filename), "bla");
+  BOOST_CHECK_EQUAL(votca::tools::filesystem::GetFileBase(filename), "bla");
 
   std::string filename2 = "blubb";
-  BOOST_CHECK_EQUAL(filesystem::GetFileBase(filename2), "blubb");
+  BOOST_CHECK_EQUAL(votca::tools::filesystem::GetFileBase(filename2), "blubb");
 
   std::string filename3 = "a.b.c.d.gro";
-  BOOST_CHECK_EQUAL(filesystem::GetFileBase(filename3), "a.b.c.d");
+  BOOST_CHECK_EQUAL(votca::tools::filesystem::GetFileBase(filename3),
+                    "a.b.c.d");
 }
 
 BOOST_AUTO_TEST_SUITE_END()

